configs/rotated_atss/metafile.yml

```diff
@@ -0,0 +1,51 @@
+Collections:
+- Name: rotated_atss
+  Metadata:
+    Training Data: DOTAv1.0
+    Training Techniques:
+      - SGD with Momentum
+      - Weight Decay
+    Training Resources: 1x GeForce GTX 1080 Ti
+    Architecture:
+      - ResNet
+  Paper:
+    URL: https://arxiv.org/abs/1912.02424
+    Title: 'Bridging the gap between anchor-based and anchor-free detection via adaptive training sample selection'
+  README: configs/rotated_atss/README.md
+
+Models:
+  - Name: rotated_atss_hbb_r50_fpn_1x_dota_oc
+    In Collection: rotated_atss
+    Config: configs/rotated_atss/rotated_atss_hbb_r50_fpn_1x_dota_oc.py
+    Metadata:
+      Training Data: DOTAv1.0
+    Results:
+      - Task: Oriented Object Detection
+        Dataset: DOTAv1.0
+        Metrics:
+          mAP: 65.59
+    Weights: https://download.openmmlab.com/mmrotate/v0.1.0/rotated_atss/rotated_atss_hbb_r50_fpn_1x_dota_oc/rotated_atss_hbb_r50_fpn_1x_dota_oc-eaa94033.pth
```

Comment:
v0.2.0

Reply:
okay, i will give you the model and log, then fix it according to your final feedback.

Reply:
Still v0.1.0